microsoft/testsuites/ltp/ltp.py

```diff
@@ -53,29 +55,73 @@ class Ltp(Tool):
     DEFAULT_LTP_TESTS_GIT_TAG = "20230929"
     LTP_GIT_URL = "https://github.com/linux-test-project/ltp.git"
     BUILD_REQUIRED_DISK_SIZE_IN_GB = 2
-    LTP_RESULT_PATH = "/opt/ltp/ltp-results.log"
-    LTP_OUTPUT_PATH = "/opt/ltp/ltp-output.log"
-    LTP_SKIP_FILE = "/opt/ltp/skipfile"
     COMPILE_TIMEOUT = 1800
     RUN_TIMEOUT = 12000
 
     @property
     def command(self) -> str:
-        return "/opt/ltp/runltp"
+        if self._ltp_tar_file:
+            return "/tmp/ltp/runltp"
+        else:
+            return "/opt/ltp/runltp"
+
+    @property
+    def command_dir(self) -> str:
+        if self._ltp_tar_file:
+            return "/tmp/ltp/"
```

Comment:
define the command path as a parameter, instead of set it by the tar file path.

Reply:
The change is done, thanks